tests/api/test_schema.py

```diff
@@ -80,59 +80,59 @@ def test_schema_response(schema_app):
     request = app_client.get(
         "/v1.0/test_schema/response/object/valid", headers={}, data=None
     )  # type: flask.Response
-    assert request.status_code == 200
+    assert request.status_code == 200, request.text
```

Comment:
These tests were failing during development and don't provide a lot of insight into what causes the error. Logging the request text if the assert fails makes it easier to debug.